store/interfaces.json

```diff
@@ -40,7 +40,7 @@
     "allowed": "*"
   },
   "order-title": {
-    "component": "OrderTitle"
+    "component": "OrderNumber"
```

Comment:
Why did you rename it? If you are really going to rename it you should also change the test name file and this interface.

Reply:
Oooops, nice catch!

Reply:
Done